xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPRendererOpenGLES.cpp

```diff
@@ -21,13 +21,10 @@
 #include "RPRendererOpenGLES.h"
 #include "cores/RetroPlayer/rendering/RenderContext.h"
 #include "cores/RetroPlayer/rendering/RenderVideoSettings.h"
+#include "guilib/GraphicContext.h"
```

Comment:
Dependency on guilib is not allowed in this file.

If you need a function, wrap it in CRenderContext.

If you need a simple type, move it to a separate GraphicsTypes.h file and include that file.

Reply:
This was dumb of me, sorry :disappointed: I should have saw that